### ISSUE TITLE
POP3 LOGIN-DELAY support

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7470]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#94|
|Type|enhancement|
|Created|2003-07-16 06:31:34Z|

<details><summary>Searchable metadata</summary>

```
trac-id__94 94
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1058337094000000 1058337094000000
changetime__1466547877962317 1466547877962317
version__ 
owner__ 
cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
as described by rfc 2449

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph commented|
|-|-|

```
#!html
<pre>
hooray it is a feature

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
This should be well supported by the protocol now.  t.mail
still does not do anything with it.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 0de84dd0c2e76ccbfe5377aaf22412b7ed5dda7a
```
#!CommitTicketReference repository="" revision="0de84dd0c2e76ccbfe5377aaf22412b7ed5dda7a"
Merge pull request #94 from rodrigc/8378-rodrigc-assert


Author: rodrigc
Reviewer: wsanchez
Fixes: #8378
```
